README.md

```diff
@@ -1,105 +1,34 @@
-<p align="center">
-  <a href="https://github.com/actions/typescript-action/actions"><img alt="typescript-action status" src="https://github.com/actions/typescript-action/workflows/build-test/badge.svg"></a>
-</p>
+# Terraform Register Module Action
```

Comment:
To answer my own question:

https://www.terraform.io/docs/cloud/api/modules.html#publish-a-private-module-from-a-vcs

So I think `publish-terraform-cloud-module-action`

You can probably replace a lot of uses of "register" with "publish." Or in some cases it might be better to use the noun for types rather than `${verb}Options`, e.g. `function publish(module: Module) {`. Or maybe `ModuleRequest` if you need to return a `Module`.

Reply:
sounds good, i'll update the repo name too